pallet/account-migration/src/lib.rs

```diff
@@ -121,9 +127,26 @@ pub mod pallet {
 	pub type Vestings<T: Config> =
 		StorageMap<_, Blake2_128Concat, AccountId32, Vec<VestingInfo<Balance, BlockNumber>>>;
 
+	/// [`darwinia_deposit::Deposits`] data.
+	#[pallet::storage]
+	#[pallet::unbounded]
+	#[pallet::getter(fn deposit_of)]
+	pub type Deposits<T: Config> = StorageMap<_, Blake2_128Concat, AccountId32, Vec<Deposit>>;
+
+	/// [`darwinia_staking::Bonded`] data.
+	///
+	/// <https://github.dev/darwinia-network/darwinia-common/blob/6a9392cfb9fe2c99b1c2b47d0c36125d61991bb7/frame/staking/src/lib.rs#L592>
```

Comment:
Suggest changing `6a9392cfb9fe2c99b1c2b47d0c36125d61991bb7` to branch, we can't tell the code version info from this commit.